docs/src/main/asciidoc/resteasy-reactive.adoc

```diff
@@ -1812,3 +1812,19 @@ public class ResourceForApp1Only {
 ----
 
 Please note that if a JAX-RS Application has been detected and the method `getClasses()` and/or `getSingletons()` has/have been overridden, Quarkus will ignore the build time conditions and consider only what has been defined in the JAX-RS Application.
+
+
+== RESTEasy Reactive client
+In addition to the Server side, RESTEasy Reactive comes with a new MicroProfile Rest Client implementation that is non-blocking at its core.
+
+To use it, add the `quarkus-rest-client-microprofile` extension to your project.
+The client can be used as described in https://quarkus.io/guides/rest-client[Using the REST Client guide] and specified in the https://download.eclipse.org/microprofile/microprofile-rest-client-1.4.1/microprofile-rest-client-1.4.1.html[MicroProfile Rest Client specification], with a few exceptions:
+
+- the extension name
+- the default scope of the client for the new extension is `@ApplicationScoped` while the `quarkus-rest-client` defaults to `@Dependent`
```

Comment:
According to `ReactiveResteasyMpClientConfig` the default is `javax.enterprise.context.SessionScoped`.

Reply:
oops, ApplicationScoped is correct, updating. Good catch!